core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -237,7 +241,7 @@ export class TranslateTextService {
       imagesData,
       dataFormat
     ).then(() => {
-      this.stateAndContent[this.activeIndex].status = this.SUBMITTED;
+      this.stateAndContent[this.activeIndex].status = this.SUBMITTED as Status;
```

Comment:
Update the type of `SUBMITTED` and `PENDING` at time of declaration to `Status` instead.

Reply:
Done.